console/src/main/java/org/matrix/console/Matrix.java

```diff
@@ -207,7 +208,7 @@ public void refreshPushRules() {
      */
     public synchronized void clearSession(Context context, MXSession session, Boolean clearCredentials) {
         if (clearCredentials) {
-            mLoginStorage.removeCredentials(session.getCredentials());
+            mLoginStorage.removeCredentials(session.getHomeserverConfig());
```

Comment:
Nit: Feels like a method name mismatch here. I'm removing credentials but passing it... a config? Same comment for `:239` below.